apps/storybook/src/forms/Switch.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { Switch } from "@optiaxiom/react";
+
+const meta: Meta<typeof Switch> = {
+  args: {
+    id: "switch-1",
```

Comment:
passing an `id` should not be required - the component should generate its own `id` if one is not given